azurelinuxagent/ga/confidential_vm_util.py

```diff
@@ -0,0 +1,46 @@
+# Windows Azure Linux Agent
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.6+ and Openssl 1.0+
+#
+
+from azurelinuxagent.common.protocol.imds import get_imds_client
+from azurelinuxagent.common import logger
+from azurelinuxagent.common import event
+from azurelinuxagent.common.future import ustr
+
```

Comment:
Please add a comment that this is just temporary (telemetry release?) 

It will help in order to review some behavior, like how should be handle errors trying to reach IMDS